app/presenters/two_factor_authentication/sign_in_phone_selection_presenter.rb

```diff
@@ -0,0 +1,55 @@
+module TwoFactorAuthentication
+  class SignInPhoneSelectionPresenter < SignInSelectionPresenter
+    attr_reader :configuration, :user, :method
+
+    def initialize(user:, configuration:, method:)
+      @user = user
+      @configuration = configuration
+      @method = method
+    end
+
+    def type
+      if MfaContext.new(configuration&.user).phone_configurations.many?
+        "#{super}_#{configuration.id}"
+      else
+        super
+      end
+    end
+
+    def label
+      if method.to_s == 'sms'
+        t('two_factor_authentication.login_options.sms')
+      elsif method.to_s == 'voice'
+        t('two_factor_authentication.login_options.voice')
+      end
+    end
+
+    def info
+      case method.to_s
+      when 'sms'
+        t(
+          'two_factor_authentication.login_options.sms_info_html',
+          phone: configuration.masked_phone,
+        )
+      when 'voice'
+        t(
+          'two_factor_authentication.login_options.voice_info_html',
+          phone: configuration.masked_phone,
+        )
+      else
+        t('two_factor_authentication.two_factor_choice_options.phone_info')
+      end
+    end
+
+    def disabled?
+      case method.to_s
+      when 'sms'
+        OutageStatus.new.vendor_outage?(:sms)
+      when 'voice'
```

Comment:
Minor: I'd wonder if it would just be simpler to match on the symbol type rather than converting it to a string?

```suggestion
      case method
      when :sms
        t(
          'two_factor_authentication.login_options.sms_info_html',
          phone: configuration.masked_phone,
        )
      when :voice
        t(
          'two_factor_authentication.login_options.voice_info_html',
          phone: configuration.masked_phone,
        )
      else
        t('two_factor_authentication.two_factor_choice_options.phone_info')
      end
    end

    def disabled?
      case method
      when :sms
        OutageStatus.new.vendor_outage?(:sms)
      when :voice
```